pkg/resources/management.cattle.io/v3/projectroletemplatebinding/ProjectRoleTemplateBinding.md

```diff
@@ -0,0 +1,28 @@
+## Validation Checks
+
+### Escalation Prevention
+
+Users can only create/update ProjectRoleTemplateBindings with rights less than or equal to those they currently possess. This is to prevents privilege escalation. 
+
+### Invalid Fields - Create
+
+Users cannot create ProjectRoleTemplateBindings which violate the following constraints:
+- Either a user subject (through "UserName" or "UserPrincipalName") or a group subject (through "GroupName" or "GroupPrincipalName"), but not both, must be specified
```

Comment:
Same comment as ClusterRoleTemplateBindings. Could maybe use some clearer language.

Reply:
Made a change here (and in the PRTB one) feel free to un-resolve if you disagree with it.

Reply:
Made a change here (and in the CRTB one) feel free to un-resolve if you disagree with it.